lib/fog/compute/google/models/instance_group_managers.rb

```diff
@@ -0,0 +1,22 @@
+module Fog
+  module Compute
+    class Google
+      class InstanceGroupManagers < Fog::Collection
+        model Fog::Compute::Google::InstanceGroupManager
+
+        def all(filters = {})
+          if filters[:zone]
+            data = Array(service.list_aggregated_instance_group_managers(filters[:zone]))
+          else
+            data = []
+            service.list_aggregated_instance_group_managers.items.each_value do |group|
```

Comment:
I believe this could be simplified to 
```ruby
data = service.list_aggregated_instance_group_managers.items.map {|group| group.instance_group_managers }
```

Reply:
This is not written like that in others files, but yes, we can do that.